<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>themkhoanchi_form</class>
 <widget class="QMainWindow" name="themkhoanchi_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm khoản chi</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_chitieu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string>Danh mục chi tiêu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sotien">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Số tiền</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_tien">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ngay">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ngày</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ghichu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ghi chú</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ghichu">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_update">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Cập nhật</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Hủy bỏ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="tree_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Chi phí cố định</string>
     </property>
     <item>
      <property name="text">
       <string>Nhà cửa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hóa đơn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Người thân</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Chi phí sinh hoạt</string>
     </property>
     <item>
      <property name="text">
       <string>Chợ </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ăn uống</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Di chuyển</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Chi phí phát sinh</string>
     </property>
     <item>
      <property name="text">
       <string>Mua sắm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Giải trí</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Làm đẹp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sức khỏe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Từ thiện</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Các khoản khác</string>
     </property>
     <item>
      <property name="text">
       <string>Đầu tư</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiết kiệm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Học tập</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Khác</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QDateEdit" name="edit_ngay">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>9999</year>
      <month>12</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2025</year>
      <month>3</month>
      <day>15</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string>Mục đích chi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Outlook</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ic_return.png</normaloff>../ic_return.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
